pkg/registry/storage/rest/storage_storage.go

```diff
@@ -64,10 +72,13 @@ func (p RESTStorageProvider) v1alpha1Storage(apiResourceConfigSource serverstora
 	return storage
 }
 
-func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) map[string]rest.Storage {
+func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (map[string]rest.Storage, error) {
 	storage := map[string]rest.Storage{}
 	// storageclasses
-	storageClassStorage := storageclassstore.NewREST(restOptionsGetter)
+	storageClassStorage, err := storageclassstore.NewREST(restOptionsGetter)
+	if err != nil {
+		return storage, err
+	}
 	storage["storageclasses"] = storageClassStorage
 
 	// volumeattachments
```

Comment:
do the NewStorage calls below also suppress panics?

Reply:
Handled now